enclave/core/sgx/linux/threadlocal.c

```diff
@@ -201,8 +203,8 @@ static volatile uint64_t _tbss_align = 1;
 static volatile bool _thread_locals_relocated = false;
 
 /* Thread local variables to track functions to call on thread exit */
-static __thread oe_tls_atexit_t* _tls_atexit_functions;
-static __thread uint64_t _num_tls_atexit_functions;
+static OE_THREAD_LOCAL_STORAGE oe_tls_atexit_t* _tls_atexit_functions;
```

Comment:
Why do we need to change SGX specific thread-local implementation if TrustZone does not support TLS?

Reply:
Technically we don't. I figured that since I changed `__thread` to `OE_THREAD_LOCAL_STORAGE` everywhere else, for consistency I'd so here too.

Having said that, I see now that it could make the reader think that TLS is sometimes not supported or could be disabled on SGX, which is certainly not the case.

What do you think? I'll happily revert this change.

Reply:
I'd prefer reverting this change.

Reply:
Will do.